lib/normalizeEmail.js

```diff
@@ -129,6 +132,11 @@ function normalizeEmail(email, options) {
     if (options.all_lowercase || options.yahoo_lowercase) {
       parts[0] = parts[0].toLowerCase();
     }
+  } else if (~yandex_domains.indexOf(parts[1])) {
+    if (options.all_lowercase || options.yahoo_lowercase) {
```

Comment:
Hi there! Thanks for the PR :)
I think it shouldn't use `options.yahoo_lowercase` because this option is for `yahoo` case only